theme/static/js/custom.js

```diff
@@ -173,6 +173,19 @@ $(document).ready(function () {
     $("#universalMessage a").on('click', function() {
         $("#universalMessage").slideUp();
         return false
+    });
+
+    $.ajax({
+        url: "/hsapi/userInfo/",
+        success: function(user) {
+            if(!user.title || !user.organization) {
+                var message = 'Your profile is nearly complete. ';
+                message += 'Please fill in your <strong>title</strong> and <strong>organization</strong>';
+                message += 'on the <a href="/user/'+user.id+'/">user profile</a> page'
+                showUniversalMessage("warn", message, 10000)();
```

Comment:
@aphelionz It'd be good to notify users whether the title only or the organization only or both need to be filled out.